packages/automated-dispute/src/services/eboActor.ts

```diff
@@ -344,11 +378,20 @@ export class EboActor {
 
         if (response) {
             await this.finalizeRequest(request, response);
-
             return;
         } else {
-            // TODO: check for responseModuleData.deadline, if no answer has been accepted after the deadline
-            //  notify and (TBD) finalize with no response
+            // Check if the response deadline has passed with no accepted responses
```

Comment:
I think we might have an edge case where there are no finalizable responses (yet), there are still some open disputes and one of them could end up with `Lost` status, thus, causing the disputed response to be the accepted one.

We should check for active disputes, prior finalizing with no request. Something like this probably?
```suggestion
            const activeDisputes = this.getActiveDisputes();
            if (activeDisputes.length === 0) return;
            
            // Check if the response deadline has passed with no accepted responses
```